config/webpack.config.js

```diff
@@ -540,8 +460,7 @@ module.exports = function(webpackEnv) {
         new MiniCssExtractPlugin({
           // Options similar to the same options in webpackOptions.output
           // both options are optional
-          filename: 'static/css/[name].[contenthash:8].css',
```

Comment:
if we stop giving hashes to the css name, how will the browser know not to pick the css file in the cache but pick the one  in the server?

Reply:
The browser will have to hit the server for the .css & .js files. I mentioned under `notes` that we are optimizing for ease of integration into portal.